controller/api_test.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/fission/fission"
 	"github.com/fission/fission/controller/client"
 	"github.com/fission/fission/crd"
+	"reflect"
```

Comment:
nit: Would be good to move this with other system-level imports instead with Fission imports